CMSgov/onemac
# Oy2 21928 - update FAQ for package view

Story: https://qmacbis.atlassian.net/browse/OY2-21928
Endpoint: See github-actions bot comment

### Details

Updates for Package View release. New onboarding materials. New Q&A section.

### Changes

- Created a VideoContent component that will embed a video and provide optional open/close section for a transcript
- Replaced content of links in Onboarding section of FAQ for OneMAC State Submitter Guide and OneMAC CMS User Guide
- Removed link in Onboarding section of FAQ for CMS Role Approver guide
- Added embedded video and transcript (using VideoContent component) for OneMAC State System Overview
- Moved faqContent in libs to its own subfolder named faq: ui-src/src/libs/faq
- created /libs/faq/stateSystemOverviewTranscript.tsx to support faqContent

### Test Plan

1. Navigate to FAQ page (no login necessary)
2. Expand Onboarding section
3. Verify content of OneMAC State Submitter Guide and OneMAC CMS User Guide
4. Verify content is removed for CMS Role Approver guide
5. Verify new section named "OneMAC State System Overview" appears below list of onboarding links
6. Verify video appears in this new section - can play; fullscreen; has CC
7. Verify link under video to "Show Transcript" and when clicked expands with transcript content; when clicked again hides transcript content
